docs/static/02-tutorials/slurm-accounting/pcluster/cluster-config.yaml

```diff
@@ -1,39 +1,26 @@
-Region: eu-west-1
-Image:
-  Os: alinux2
 HeadNode:
-  InstanceType: t3.xlarge
+  InstanceType: c5.xlarge
   Networking:
-    SubnetId: subnet-03ffbdaeee8264db7
-    AdditionalSecurityGroups:
-      - sg-0fec342c8a6f2a5ea
-  Ssh:
-    KeyName: aws_rsa
+    SubnetId: subnet-846f1aff
```

Comment:
seems like we shouldn't have this magic number in the config

Reply:
agreed